util.go

```diff
@@ -38,7 +38,7 @@ func encodeQuery(query map[string][]string) string {
 			continue
 		}
 
-		tuple := key + "=" + oDataEncodeVals(vals)
+		tuple := key + "=" + strings.Replace(strings.Join(vals, ","), " ", "%20", -1)
```

Comment:
All I know is that if I have a query such as `user/any(u:((tolower(u/username)) eq ('wrboyce')))` then prior to my changes the `/():` characters would be encoded and this, seemingly, was causing the request to fail. Perhaps a specific list of characters which should not be encoded would be a better approach.